source/Outfit.cpp

```diff
@@ -189,16 +189,21 @@ void Outfit::Load(const DataNode &node)
 	if(node.Size() >= 2)
 	{
 		name = node.Token(1);
-		pluralName = name + 's';
 	}
 	isDefined = true;
 
+	bool hasPluralName = false;
```

Comment:
Isn't this redundant with pluralName.empty() since we aren't setting the pluralName when the true name is set?
It also looks like we aren't setting the display name to the true name if no display name is given. So I'd expect something like the following to be done:
```c++
if(node.Size() >= 2)
	name = node.Token(1);

for(const DataNode &child : node)
{
	if(child.Token(0) == "display name" && child.Size() >= 2)
		displayName = child.Token(1);
	...
	else if(child.Token(0) == "plural" && child.Size() >= 2)
		pluralName = child.Token(1);
	...
}

if(displayName.empty())
	displayName = name;
if(pluralName.empty())
	pluralName = displayName + 's';

Reply:
done